src/frameworks/drupal8/solr.md

```diff
@@ -4,6 +4,15 @@ Platform.sh recommends using Solr version 6.6, yet we support [other versions](.
 
 Advanced Solr service configuration and implementation in other frameworks other than Drupal can be found at the [Solr services page](../../configuration/services/solr.md)
 
+> **note**
+> Before Installing or Upgrading
+>
+> Recent changes to the module [Search API Solr](https://www.drupal.org/project/search_api_solr) introduced a bug which may prevent the Solr server from starting properly. As a consequence, the deployment may fail. Then, we highly recommend testing on development branches first.
+>
+> This is caused by having a wrong value for `solr.install.dir`defined in the configuration file `solrcore.properties` provided by the module's templates.
+>
+> The fix is to remove the line defining `solr.install.dir` in `solrcore.properties`. However, in some cases which the Solr server startup command does not define this value, then it may be necessary to set this value to `solr.install.dir=/opt/solr/<solr_version>`, for example, `solr.install.dir=/opt/solr/6.6`.
```

Comment:
This is good point-in-time documentation.  However, anyone setting up Solr now is going to want just this new version, not the old version at all.

Rather, the page should be restructured to show the current-version instructions first, followed by "for Search API Solr before version X, these things are different.  Please upgrade but test in a branch because it's a bumpy ride."